### DISCUSSION TITLE
Cron, not working in the repo.

### DISCUSSION THREAD

**Discussion by shinegami-2002** [brianhhuynh38/slackpoint-v3#14] [General]
Shall we drop cron as it is not working properly, what's ur take on this shall we keep it or drop it?

**Comment by brianhhuynh38** [comment]
I put this in the GitHub Issues already, actually. It doesn't seem to function like it does in the gif that the ReadMe has. I've removed it from the ReadMe since it doesn't reflect the actual functionality of the program. We could fix it, but we already have our hands too full trying to get our own functionalities to work.

Additionally, the test for this function literally just says "YOLO" and doesn't really check anything as far as I'm aware. Probably should mention it as a possible thing for the next team to do.

**Comment by shinegami-2002** [comment]
We are dropping it then, since it dosen't seem to work
